zb-db/src/main/java/io/camunda/zeebe/db/impl/rocksdb/transaction/TransactionalColumnFamily.java

```diff
@@ -186,27 +207,32 @@ public void deleteExisting(final KeyType key) {
 
   @Override
   public boolean exists(final KeyType key) {
-    columnFamilyContext.wrapValueView(new byte[0]);
```

Comment:
🤔 Any idea why it wrapped this before? ❓ Is it really no longer necessary?

Reply:
Good point, I sort of forgot about this change.
I believe this was unnecessary, even before my changes here, because `getValue` would always call `columnFamilyContext.wrapValueView(...)` anyway which always wraps the value view, either around the byte array from RocksDB or around an empty byte array when RocksDB finds nothing.

Reply:
This would only be necessary if `ensureInOpenTransaction` could silently fail. Then without manually wrapping around an empty byte array, we would return the wrong result because `columnFamilyContext.wrapValueView(...)` was never called.